src/map/battle.cpp

```diff
@@ -5965,22 +6087,38 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 						break;
 #ifdef RENEWAL
 					case WZ_HEAVENDRIVE:
+						skillratio += -100 + skill_lv * 100 + skill_lv * 25;
+						break;
 					case WZ_METEOR:
 						skillratio += 25;
 						break;
 					case WZ_VERMILION:
-						if(sd) {
-							int per = 0;
-							while ((++per) < skill_lv)
-								skillratio += per * 5; //100% 105% 115% 130% 150% 175% 205% 240% 280% 325%
-						} else {
+						if(sd)
+							skillratio += 25 + skill_lv * 5;
```

Comment:
according to https://www.divine-pride.net/database/skill/85/lord-of-vermilion (KRO page)
```suggestion
							skillratio += 300 + skill_lv * 100;
```

Reply:
Earthspike have upgrade damage to 200% matk.
from here https://www.divine-pride.net/database/skill/90/ (KRO page)

it should have some case around here like (skillratio *= 2)

Reply:
LoV is like that because it's each individual hit. If we adjust it to your suggestion then then LoV level 10 would be: (400 + 10 * 100) *10 hits.

Earth Spike's damage increase is in this PR already as:
```cpp
#ifdef RENEWAL
					case WZ_EARTHSPIKE:
						skillratio += 100;
						break;
#endif
```